app/http.php

```diff
@@ -35,6 +35,7 @@
 use Utopia\Http\Route;
 use Utopia\Http\Validator\Assoc;
 use Utopia\Http\Validator\Boolean;
+use Utopia\Http\Validator\FloatValidator;
```

Comment:
Can we rename it to `Float`? It's already in namespace of `Validator`.

Reply:
its actually called FloatValidator in the http library 
https://github.com/utopia-php/http/blob/master/src/Http/Validator/FloatValidator.php

We need to fix it there. 

For now we can do `use Utopia\Http\Validator\FloatValidator as Float;`

Reply:
We can't do either (I wanted to do this previously 😅) Class names are case-insensitive in PHP, and classes named 'int', 'string', 'float', 'bool', 'true', 'false', or 'null' are forbidden